src/libxively/sft/xi_sft_logic.c

```diff
@@ -372,71 +294,22 @@ xi_sft_on_message( xi_sft_context_t* context, xi_control_message_t* sft_message_
                             // downloaded but file close error: abort or continue?
                             // goto err_handling;
```

Comment:
According to the [man page for `close`](http://man7.org/linux/man-pages/man2/close.2.html):

```
A careful programmer will check the return value of close(), since it
       is quite possible that errors on a previous write(2) operation are
       reported only on the final close() that releases the open file
       description.  Failing to check the return value when closing a file
       may lead to silent loss of data.
```

Given that, I suggest aborting here instead of continuing.

Reply:
ok, thx JC doing it